src/main/java/com/digirati/taxonomy/manager/normalisation/TextNormaliser.java

```diff
@@ -0,0 +1,80 @@
+package com.digirati.taxonomy.manager.normalisation;
+
+import edu.stanford.nlp.ling.CoreLabel;
+import edu.stanford.nlp.pipeline.CoreDocument;
+import edu.stanford.nlp.pipeline.StanfordCoreNLP;
+
+import java.util.Properties;
+import java.util.Set;
+
+/**
+ * Provides a mechanism for normalising any text inputs, including the terms to search for and the
+ * text in which to search.
+ */
+public class TextNormaliser {
+
+    private final Set<String> stopwords;
+
+    private StanfordCoreNLP pipeline;
```

Comment:
Should still be `final`.

Reply:
This can't be final as things currently stand - it gets set in `initialiseNormaliser`. I'll make sure to move it back into the constructor and mark it as final when I action the point about that method though.